<!-- -->
<launch>
  <!--<node name="astar_planner" cwd="node" pkg="astar_planner" type="astar" required="true" output="screen"> -->
    <node pkg="a_start" type="a_star" name="a_star" required="true" output="screen">
    <!-- fixed parameter -->
    <!-- map size and resolution -->
    <param name="map_width" type="double" value="6.44"/>
    <param name="map_height" type="double" value="3.33"/>
    <param name="map_resolution" type="double" value="0.01"/>

    <!-- tunable parameter -->
    <!-- grid resolution -->
    <param name="grid_resolution" type="double" value="0.1"/>

    <!-- start position and goal position -->
    <param name="startx" type="double" value="-3.15"/>
    <param name="starty" type="double" value="1.40"/>
    <param name="goalx" type="double" value="-1.50"/>
    <param name="goaly" type="double" value="-1.55"/>
    <!--<rosparam file="$(find a_start)/data/nodes.yaml" command="load" /> -->
    <rosparam file="$(find a_start)/data/topic_list.yaml" command="load" />
  </node>

  <node pkg="a_start" type="visualiser" name="visualiser" required="true" output="screen">
  <rosparam file="$(find a_start)/data/landmarks.yaml" command="load" />
  <rosparam file="$(find a_start)/data/topic_list.yaml" command="load" />
  </node>

  <node pkg="a_start" type="main_path" name="main_path" required="true" output="screen" >
  <rosparam file="$(find a_start)/data/landmarks.yaml" command="load" />
  <rosparam file="$(find a_start)/data/topic_list.yaml" command="load" />
  <rosparam file="$(find a_start)/data/map.yaml" command="load" />
  </node>

 <!--<node pkg="map_server" type="map_server" name="map_server" required="true" args="$(find a_start)/data/map.yaml" /> -->
  <!-- <node pkg="rviz" type="rviz" name="rviz" required="true" /> -->
</launch>

